rust/src/crypto/sign.rs

```diff
@@ -119,33 +124,15 @@ impl ByteObject for Signature {
     }
 }
 
-impl Signature {
-    /// Compute the floating point representation of the hashed signature and ensure that it
-    /// is below the given threshold: int(hash(signature)) / (2**hashbits - 1) <= threshold.
-    pub fn is_eligible(&self, threshold: f64) -> bool {
```

Comment:
I initially wanted to keep business logic out of the `crypto` module: the crypto module would just provide business logic agnostic types and helpers.  So I moved this out to `crate::mask`. But then changed my mind and put it back to `crate::crypto` but in another module.

I can put it back where it was originally but what's your thought about what should be part of `crate::crypto` and `crate::mask`? Does the "business concerns" separation make sense, and where is the boundary between pure crypto stuff and business logic?

Reply:
i agree that it makes sense to have a separation between business logic and helper layers here. but i find it quite hard to put this example in one of the categories, because this method in itself doesn't bear any business logic meaning in itself since it just compares a hash on a signature which can be considered crypto logic, but it only becomes business logic when we interpret the result of this calculation as such.

long story short, the name is to some extend business logic whereas the body is crypto logic, maybe we have to split/rename this to have a better separation.

Reply:
Yeah I agree with that.

> maybe we have to split/rename this to have a better separation.

I also agree. In the meantime though, do you prefer me to move it back to `crate::crypto`?

Reply:
i would prefer to move it back to the `crypto` module. if we split it later, the body will probably stay there and only the name will change to be wrapped with a different name somewhere else.

Reply:
done